packages/frontend/src/lib/store.ts

```diff
@@ -0,0 +1,11 @@
+import { configureStore } from '@reduxjs/toolkit';
+
+export const store = () => {
```

Comment:
**suggestion:** since a store is created per request we should call this `makeStore`, so that we know to call it as a function.

Reply:
Thanks! I've updated the function name to makeStore.